logger/t/test_config.cc

```diff
@@ -362,13 +397,16 @@ TEST_F(ConfigTest, TableNameValidation_DefaultNameValidation)
 	auto config_path = temp_dir / "default_table.json";
 	write_config(config_path, R"({
 		"access_log": {
-			"host": "test.com"
+			"plugin_path": "./mmap_plugin.so",
+			"clickhouse": {
+				"host": "test.com"
+			}
 		}
 	})");
 
 	auto config = TfwLoggerConfig::load_from_file(config_path);
 	ASSERT_TRUE(config.has_value());
 	EXPECT_NO_THROW(config->validate())
 		<< "Default table name should be valid";
-	EXPECT_EQ(config->clickhouse.table_name, "access_log");
+	EXPECT_EQ(config->clickhouse_mmap->table_name, "access_log");
```

Comment:
Please also add new tests: no path to a plugin or wrong path (we should correctly terminate with an adequate message), maybe something mode